tests/xtro-sharpie/SimdCheck.cs

```diff
@@ -45,6 +45,7 @@ class NativeSimdInfo
 			{ "vector_uint4", new NativeSimdInfo { Managed = "Vector4i", }},
 			// simd_floatX is typedefed to vector_floatX
 			{ "simd_float2", new NativeSimdInfo { Managed = "Vector2" }},
+			{ "NS_RETURNS_INNER_POINTER const simd_float2", new NativeSimdInfo { Managed = "Vector2" }},
```

Comment:
Adding `NS_RETURNS_INNER_POINTER` to the Undecorate function didn't work: https://github.com/xamarin/xamarin-macios/pull/9523#discussion_r478168664?

Reply:
Definitely today not at my best... yep you are right... 🤦 2x